assets/js/dintero-checkout-express.js

```diff
@@ -386,18 +385,55 @@ jQuery( function( $ ) {
 		},
 
 		/**
-		 * Submit the order using the WooCommerce AJAX function.
-		 *
-		 * @param {callback} callback
+		 * Block form fields from being modified by the user.
 		 */
-		submitOrder( callback ) {
+		blockForm() {
+			/* Order review. */
 			$( '.woocommerce-checkout-review-order-table' ).block( {
+				message: dinteroCheckoutParams.pip_text,
```

Comment:
can you include a screenshot of how the form will look when blocked with this change?

Reply:
@svnv Any feedback to this?

Reply:
Added a suggestion with some added padding and border radius the same as in the checkout.